### ISSUE TITLE
How to not duplicate values of input fields while adding a new dynamic form?

### ISSUE THREAD

**Issue by vitalypark0555** [kidzen/yii2-dynamicform#20]
When I fill all the fields of the first dynamic form, adding a new one all fields are filled in it by the values of the former one. How can I create just empty new form?

**Comment by kidzen** [comment]
Please ensure the field are mark with indexes ({$i}) for dynamic form field..
```php
   ...
    <?= $form->field($asset, "[{$i}]category_id")->label(Yii::t('app', 'Category'))
   ...
```

**Comment by kidzen** [comment]
please do composer update...i have mistakenly publish dev branch...tq for your support

**Comment by vitalypark0555** [comment]
Good day, kidzen.
Why namespace is wbraganca/dynamicform if in the instruction you wrote like kidzen/dynamicform

**Comment by kidzen** [comment]
indeed, this repo originally from wbraganca/dynamicform in which now im maintaining on my own repo since the owner no longer active..
could you point out any mistake if i miss out any namespace refactor?

**Comment by masgeek** [comment]
This issue is still present in the dev/master branch
